app/views/kudos/_kudos.html.erb

```diff
@@ -9,19 +9,19 @@
               html_safe %>
         <% else %>
           <%= kudos[0,ArchiveConfig.MAX_KUDOS_TO_SHOW].map {|kudo| link_to kudo.pseud.byline, kudo.pseud.user}.
-              to_sentence(:last_word_connector => ', ').
-              html_safe + ', ' %>
-          <%= link_to work_kudos_path(commentable), :id => "kudos_summary" do %>
+              to_sentence(last_word_connector: ", ").
+              html_safe + ", " %>
+          <%= link_to work_kudos_path(commentable), id: "kudos_summary" do %>
            <% collapsed_count = kudos.length - ArchiveConfig.MAX_KUDOS_TO_SHOW %>
             <% if collapsed_count == 1 %>
-              <%= ts(' and 1 more user') %>
+              <%= ts(" and 1 more user") %>
             <% else %>
-              <%= ts(' and %{collapsed_count} more users', collapsed_count: collapsed_count) %>
+              <%= ts(" and %{collapsed_count} more users", collapsed_count: collapsed_count) %>
             <% end %>
           <% end %>
           <span class="kudos_expanded hidden">
             <%= kudos[ArchiveConfig.MAX_KUDOS_TO_SHOW..-1].map {|kudo| link_to kudo.pseud.byline, kudo.pseud.user}.
-                to_sentence(:two_words_connector => ts(" and "), :last_word_connector => ts(" and ")).
+                to_sentence(two_words_connector: ts(" and "), last_word_connector: ts(" and ")).
```

Comment:
The English default for `two_words_connector` is already " and " so I don't know why we're overriding this, but I'm amused that we're overriding Ruby's default Oxford comma.